float64[] cones_left_x
float64[] cones_left_y
float64[] cones_right_x
float64[] cones_right_y
